fatal error: too many errors emitted.

**OS:** [OSX 10.10.5]
**Version:** [0.0.8]
**Commit/Build:** [9bd4284]

<!-- Explanation of the issue -->
Hi, I have this error when try to build : 
/usr/local/include/SDL2/SDL_platform.h:73:5: error: 'TARGET_OS_TV' is not defined, evaluates to 0 [-Werror,-Wundef]
#if TARGET_OS_TV
    ^
In file included from /Users/fabrizio/github/games/OpenRCT2/src/openrct2/cheats.c:17:
In file included from /Users/fabrizio/github/games/OpenRCT2/src/openrct2/cheats.h:20:
/Users/fabrizio/github/games/OpenRCT2/src/openrct2/common.h:294:1: error: expected parameter declarator
assert_struct_size(registers, 7 * 4);
^
/Users/fabrizio/github/games/OpenRCT2/src/openrct2/common.h:227:48: note: expanded from macro 'assert_struct_size'
#define assert_struct_size(x, y) static_assert(sizeof(x) == (y), "Improper struct size")
....
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
Thanks.

With xcode project I have this error : 

Ld /Users/fabrizio/Library/Developer/Xcode/DerivedData/OpenRCT2-epbanyvebcmsrxdbyuzoabksynnx/Build/Products/Debug/OpenRCT2.app/Contents/MacOS/OpenRCT2 normal x86_64
    cd /Users/fabrizio/github/games/OpenRCT2-develop
    export MACOSX_DEPLOYMENT_TARGET=10.8
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++ -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -L/Users/fabrizio/Library/Developer/Xcode/DerivedData/OpenRCT2-epbanyvebcmsrxdbyuzoabksynnx/Build/Products/Debug -L/Users/fabrizio/github/games/OpenRCT2-develop/libxc/lib -F/Users/fabrizio/Library/Developer/Xcode/DerivedData/OpenRCT2-epbanyvebcmsrxdbyuzoabksynnx/Build/Products/Debug -filelist /Users/fabrizio/Library/Developer/Xcode/DerivedData/OpenRCT2-epbanyvebcmsrxdbyuzoabksynnx/Build/Intermediates/OpenRCT2.build/Debug/OpenRCT2.build/Objects-normal/x86_64/OpenRCT2.LinkFileList -Xlinker -no_pie -Xlinker -rpath -Xlinker @executable_path/../Frameworks -mmacosx-version-min=10.8 -stdlib=libc++ -fobjc-link-runtime -lopenrct2 -lz -lcurl -liconv -lcrypto -ljansson -lpng16 -lSDL2 -lSDL2_ttf -lspeexdsp -lzip -Xlinker -dependency_info -Xlinker /Users/fabrizio/Library/Developer/Xcode/DerivedData/OpenRCT2-epbanyvebcmsrxdbyuzoabksynnx/Build/Intermediates/OpenRCT2.build/Debug/OpenRCT2.build/Objects-normal/x86_64/OpenRCT2_dependency_info.dat -o /Users/fabrizio/Library/Developer/Xcode/DerivedData/OpenRCT2-epbanyvebcmsrxdbyuzoabksynnx/Build/Products/Debug/OpenRCT2.app/Contents/MacOS/OpenRCT2

Undefined symbols for architecture x86_64:
  "_CTFontDescriptorCopyAttribute", referenced from:
      _platform_get_font_path in libopenrct2.a(macos.o)
  "_CTFontDescriptorCreateWithNameAndSize", referenced from:
      _platform_get_font_path in libopenrct2.a(macos.o)
  "_OBJC_CLASS_$_NSAlert", referenced from:
      objc-class-ref in UiContext.macOS.o
      objc-class-ref in libopenrct2.a(macos.o)
  "_OBJC_CLASS_$_NSOpenPanel", referenced from:
      objc-class-ref in UiContext.macOS.o
  "_OBJC_CLASS_$_NSSavePanel", referenced from:
      objc-class-ref in UiContext.macOS.o
  "_OBJC_CLASS_$_NSWindow", referenced from:
      objc-class-ref in UiContext.macOS.o
      objc-class-ref in libopenrct2.a(macos.o)
  "_kCTFontURLAttribute", referenced from:
      _platform_get_font_path in libopenrct2.a(macos.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

--- a/OpenRCT2.xcodeproj/project.pbxproj
+++ b/OpenRCT2.xcodeproj/project.pbxproj
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 46;
+	objectVersion = 48;
 	objects = {
 
 /* Begin PBXAggregateTarget section */
@@ -2689,7 +2689,7 @@
 				};
 			};
 			buildConfigurationList = D497D0731C20FD52002BF46A /* Build configuration list for PBXProject "OpenRCT2" */;
-			compatibilityVersion = "Xcode 3.2";
+			compatibilityVersion = "Xcode 8.0";
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (

